lib/lita/handlers/jira_issues.rb

```diff
@@ -34,53 +36,62 @@ def handle_key(response, key)
       def issue_details(data)
         key = data[:key]
         data = data[:fields]
-        issue = summary(key, data)
-        issue << status(data)
-        issue << assignee(data)
-        issue << reporter(data)
-        issue << fix_version(data)
-        issue << priority(data)
-        issue << issue_link(key)
-      end
-
-      def summary(key, data)
-        "[#{key}] #{data[:summary]}"
+        
+        # build out the response from the configured format
+        text = String.new(config.format)
+        text.sub!('%I', key.upcase)
```

Comment:
Sounds awesome :+1:. Whenever you have time to update, I'll be happy to review. Thanks again!